tutor/src/screens/task/ux.js

```diff
@@ -278,17 +287,15 @@ export default class TaskUX {
   }
 
   @computed get previousStep() {
-    return this.canGoBackward ?
-      this.steps[this._stepIndex - 1] : null;
+    return this.steps[this._stepIndex - 1];
   }
 
   @computed get currentStepIndex() {
     return this._stepIndex;
   }
 
   @computed get nextStep() {
-    return this.canGoForward ?
-      this.steps[this._stepIndex + 1] : null;
+    return this.steps[this._stepIndex + 1];
```

Comment:
No risk of going past the last one? Same thing above.

Reply:
yeah, a slight one if the calling code doesn't call `canGoForward`, but this method isn't used anywhere outside of this class so seems safe enough.  Also `canGoForward` now calls this method, after it checks if it's safe to do so.  Without removing this you get a circle